app/src/main/java/com/vero/tasky/agenda/data/mappers/PhotoMapper.kt

```diff
@@ -5,26 +5,23 @@ import com.vero.tasky.agenda.data.local.entities.RemotePhotoEntity
 import com.vero.tasky.agenda.data.remote.network.dto.PhotoDto
 import com.vero.tasky.agenda.domain.model.AgendaPhoto
 
-object PhotoMapper {
+fun PhotoDto.toPhotoEntity(eventId: String): RemotePhotoEntity {
+    return RemotePhotoEntity(
+        key = this.key,
+        path = this.url,
+        eventId = eventId
+    )
+}
 
-    fun toPhotoEntity(eventId: String, photoDto: PhotoDto) : RemotePhotoEntity {
-        return RemotePhotoEntity(
-            key = photoDto.key,
-            path = photoDto.url,
-            eventId = eventId
-        )
-    }
+fun RemotePhotoEntity.toNetworkPhoto(): AgendaPhoto.RemotePhoto {
```

Comment:
It's "AgendaPhoto.RemotePhoto" but method is ".toNetworkPhoto"
Maybe rename it to ".toRemotePhoto"